docs/contributions/1_setup_locally.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
-title: Set up star.vote locally
-nav_order: 2000
+title: 🌱 First time Local Setup
+nav_order: 1
+parent: Contribution Guide
 ---
 
 # Set up star.vote locally
```

Comment:
I shut down our Heroku apps today, so we should switch to referencing dev.star.vote

Reply:
Pushed a pass removing the heroku references, some of the older docs will probably need some more updates though for them to be compatible with the new setup